src/transformers/models/flaubert/tokenization_flaubert.py

```diff
@@ -260,11 +260,13 @@ def __init__(
             additional_special_tokens=additional_special_tokens,
             lang2id=lang2id,
             id2lang=id2lang,
-            do_lowercase_and_remove_accent=do_lowercase_and_remove_accent,
-            do_lowercase=do_lowercase,
             **kwargs,
         )
 
+        self.do_lowercase = do_lowercase
```

Comment:
This is never initialized in the parent classes, before/after PR #19330.

Before #19330, we needed to set it just like here.